### PR TITLE
Show error/success message while changing log level in admin settings

### LINKED ISSUE #38220: [QA] admin GUI cannot change log level
Tested with docker run --rm -ti -v data:/mnt/data -p8080:8080 owncloud/server:10.6.0-rc3

* In the admin gui, the log level is set to 'Warnings, errors and fatal issues' per default
* admin user can change it there to e.G. 'Everything (fatal issues, errors, warnings, info, debug)'
* no confirmation message appears. BAD
* there is no save button on the admin page. OKayish?
* when navigating away, and returning to the admin page, the setting is back at 'Warnings, errors and fatal issues' BAD

Expected behaviour: the settings are saved automagically. User gets a confirmation message about the save.
  E.g. when changing the Update channel. the user gets a confirmation message in green:
image

### PR DESCRIPTION
**Pull request by AlexAndBear**
## Description
This change will show a success/error message, while you change the log level in settings->administration->general

image

## Related Issue

- Fixes https://github.com/owncloud/core/issues/38220

## Motivation and Context
This will give the user instant feedback if the operation was successfull

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Database schema changes (next release will require increase of minor version instead of patch)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests only (no source changes)

## Checklist:

- [x] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 
- [x] Changelog item, see [TEMPLATE](https://github.com/owncloud/core/blob/master/changelog/TEMPLATE)

### CODE REVIEW

File: settings/js/admin.js [hunk 1]
```diff
@@ -11,8 +11,12 @@ $(document).ready(function(){
 	});
 
 
-	$('#loglevel').change(function(){
-		$.post(OC.generateUrl('/settings/admin/log/level'), {level: $(this).val()});
+	$('#loglevel').change(function () {
```
Comment (JammingBen) [inline settings/js/admin.js:14]: I think you are missing the `{level: $(this).val()}` somehow? The log level is not being saved to my config.

File: settings/templates/panels/admin/logging.php [hunk 2]
```diff
@@ -14,17 +14,19 @@
...
-				$selected = '';
-				if ($i == $_['loglevel']):
-					$selected = 'selected="selected"';
-				endif; ?>
-					<option value='<?php p($i)?>' <?php p($selected) ?>><?php p($levelLabels[$i])?></option>
-			<?php endfor;?>
-			</select></p>
-			<br/>
+		<h2 class="app-name"><?php p($l->t('Log')); ?></h2>
+		<p><?php p($l->t('What to log')); ?> <select name='loglevel'
```
Comment (JammingBen) [inline settings/templates/panels/admin/logging.php:18]: Seems like `id='loglevel'>` is kinda off here. Or maybe a result of the auto-fix? :thinking:

**Comment by phil-davis** [comment]
@JanAckermann the CLA assistant thinks that you have not signed! Please click the buttons to sign...

**Review by JammingBen — COMMENTED** [review]
Now it works, nice! You accidentally committed changes to `.htaccess` and `Makefile` though. Otherwise it looks good to me.

**Comment by JanAckermann** [comment]
> @JanAckermann the CLA assistant thinks that you have not signed! Please click the buttons to sign...
@phil-davis signed it multiple times, just not working

**Comment by phil-davis** [comment]
> > @JanAckermann the CLA assistant thinks that you have not signed! Please click the buttons to sign...
> 
> @phil-davis signed it multiple times, just not working

https://docs.github.com/en/github/committing-changes-to-your-project/why-are-my-commits-linked-to-the-wrong-user#commits-are-not-linked-to-any-user

Maybe the commit does not have an email address that matches your email address in GitHub? Or?

**Review by JammingBen — APPROVED** [review]

**Pull request merged**